### PR TITLE
TECH-187: Create test configuration for "/{GovStackInstance}/{memberClass}/{memberCode}/{applicationCode}/listMethods" endpoint 

### PR DESCRIPTION
**Pull request by mgrochalskisoldevelo**
Created mocks and Cucumber tests for "/{GovStackInstance}/{memberClass}/{memberCode}/{applicationCode}/listMethods" endpoint

**Description**
Create test configuration for /{GovStackInstance}/{memberClass}/{memberCode}/{applicationCode}/listMethods endpoint from  https://github.com/GovStackWorkingGroup/bb-information-mediator/blob/main/api/govstack_im_service_metadata_api-0.3-swagger.json.  This should include all the steps that are pointed out in the parent ticket.

TICKET: https://govstack-global.atlassian.net/browse/TECH-187

### CODE REVIEW

**Review by KarolinaKopacz — APPROVED** [review]

**Comment by KarolinaKopacz** [comment]
LGTM !

File: examples/mock/mockoon-bb-information-mediator.json [hunk 1]
```diff
@@ -36,23 +36,52 @@
...
           "filePath": "",
           "databucketID": "",
           "sendFileAsBody": false,
-          "rules": [],
-          "rulesOperator": "OR",
+          "rules": [
+            {
+              "target": "params",
+              "modifier": "GovStackInstance",
+              "value": "^.{1,}$",
```
Comment (aleksander-reitsakas) [inline examples/mock/mockoon-bb-information-mediator.json]: It's identifier. Not arbitrary sequence

File: examples/mock/mockoon-bb-information-mediator.json [hunk 2]
```diff
@@ -36,23 +36,52 @@
...
+              "invert": false,
+              "operator": "regex"
+            },
+            {
+              "target": "params",
+              "modifier": "memberClass",
+              "value": "^.{1,}$",
```
Comment (aleksander-reitsakas) [inline examples/mock/mockoon-bb-information-mediator.json]: It's identifier

File: examples/mock/mockoon-bb-information-mediator.json [hunk 3]
```diff
@@ -36,23 +36,52 @@
...
+              "invert": false,
+              "operator": "regex"
+            },
+            {
+              "target": "params",
+              "modifier": "memberCode",
+              "value": "^.{1,}$",
```
Comment (aleksander-reitsakas) [inline examples/mock/mockoon-bb-information-mediator.json]: It's alphanumerical string

File: examples/mock/mockoon-bb-information-mediator.json [hunk 4]
```diff
@@ -36,23 +36,52 @@
...
+              "invert": false,
+              "operator": "regex"
+            },
+            {
+              "target": "params",
+              "modifier": "applicationCode",
+              "value": "^.{1,}$",
```
Comment (aleksander-reitsakas) [inline examples/mock/mockoon-bb-information-mediator.json]: It's alphanumerical string

File: examples/mock/mockoon-bb-information-mediator.json (hunk 1 above)
Comment (mgrochalskisoldevelo) [reply examples/mock/mockoon-bb-information-mediator.json]: To make sure. By identifier you mean UUID?
@aleksander-reitsakas

**Comment by mgrochalskisoldevelo** [comment]
@aleksander-reitsakas i changed those regexes. Let me know please if identifiers should allow other signs like _ or -
If you could provide an example it would be great.
Also the spec doesn't provide response in case those values are invalid.Should it be 400? As in getOpenAPI endpoint?

**Review by KarolinaKopacz — APPROVED** [review]

File: examples/mock/mockoon-bb-information-mediator.json (hunk 1 above)
Comment (aleksander-reitsakas) [reply examples/mock/mockoon-bb-information-mediator.json]: Identifier like variable name

**Review by aleksander-reitsakas — COMMENTED** [review]
@mgrochalskisoldevelo If GovStackInstance, memberClass, etc. do not match, then the behavior is undefined.
'_' is allowed. So are '-', '.' (despite my previous statement about alphanumeric content).

**Review by KarolinaKopacz — APPROVED** [review]

**Review by aleksander-reitsakas — CHANGES_REQUESTED** [review]
Pls add ':' also. Sorry for delay

**Review by KarolinaKopacz — APPROVED** [review]

**Review by aleksander-reitsakas — APPROVED** [review]

**Pull request merged**